### ISSUE TITLE
Duplicate Ranks

### ISSUE THREAD

**Issue by TurtIeSocks** [UnownHash/gohbem#13]
As discussed, Gohbem seems to be creating duplicate ranks, specifically for Koffing:

```json
{
    "great": [
        {
            "pokemon": 110,
            "form": 706,
            "cap": 50,
            "value": 1836309,
            "level": 23,
            "cp": 1479,
            "percentage": 0.95009,
            "rank": 1731,
            "capped": true
        },
        {
            "pokemon": 110,
            "form": 944,
            "cap": 50,
            "value": 1836309,
            "level": 23,
            "cp": 1479,
            "percentage": 0.95009,
            "rank": 1731,
            "capped": true
        },
        {
            "pokemon": 110,
            "form": 944,
            "cap": 50,
            "value": 1836309,
            "level": 23,
            "cp": 1479,
            "percentage": 0.95009,
            "rank": 1731,
            "capped": true
        },
        {
            "pokemon": 110,
            "form": 944,
            "cap": 50,
            "value": 1836309,
            "level": 23,
            "cp": 1479,
            "percentage": 0.95009,
            "rank": 1731,
            "capped": true
        }
    ],
    "ultra": [
        {
            "pokemon": 110,
            "form": 706,
            "cap": 50,
            "value": 4028491,
            "level": 48.5,
            "cp": 2500,
            "percentage": 0.99239,
            "rank": 13
        },
        {
            "pokemon": 110,
            "form": 706,
            "cap": 51,
            "value": 4028491,
            "level": 48.5,
            "cp": 2500,
            "percentage": 0.98497,
            "rank": 56
        },
        {
            "pokemon": 110,
            "form": 944,
            "cap": 50,
            "value": 4028491,
            "level": 48.5,
            "cp": 2500,
            "percentage": 0.99239,
            "rank": 13
        },
        {
            "pokemon": 110,
            "form": 944,
            "cap": 51,
            "value": 4028491,
            "level": 48.5,
            "cp": 2500,
            "percentage": 0.98497,
            "rank": 56
        },
        {
            "pokemon": 110,
            "form": 944,
            "cap": 50,
            "value": 4028491,
            "level": 48.5,
            "cp": 2500,
            "percentage": 0.99239,
            "rank": 13
        },
        {
            "pokemon": 110,
            "form": 944,
            "cap": 51,
            "value": 4028491,
            "level": 48.5,
            "cp": 2500,
            "percentage": 0.98497,
            "rank": 56
        },
        {
            "pokemon": 110,
            "form": 944,
            "cap": 50,
            "value": 4028491,
            "level": 48.5,
            "cp": 2500,
            "percentage": 0.99239,
            "rank": 13
        },
        {
            "pokemon": 110,
            "form": 944,
            "cap": 51,
            "value": 4028491,
            "level": 48.5,
            "cp": 2500,
            "percentage": 0.98497,
            "rank": 56
        }
    ]
}
```

**Comment by jfberry** [comment]
to recreate, it would likely help to have the input stats

**Comment by TurtIeSocks** [comment]
Welp I had an image and stats queued up but @Mygod has informed me that this has actually been fixed in the codebase already but hasn't been released. @lenisko plz

**Comment by Mygod** [comment]
Already fixed in #12.
